### ISSUE TITLE
Styling: Undo Duration Text not aligned with rest of items in the Settings Page

### ISSUE THREAD

**Issue by duckcommit** [CCExtractor/ultimate_alarm_clock#557]
### Description

Undo Duration Text not aligned with rest of items in the Settings Page. It is little bit to the left.a 

### Screenshots

![Screenshot_20240414_235626](https://github.com/CCExtractor/ultimate_alarm_clock/assets/22213675/e44470b7-920a-43bb-be98-98298ae11ee5)

**Comment by khush1yaaar** [comment]
hey, I'm proficient flutter developer, new to open source. looking for small issues. please assign me this issue.

**Comment by AryanSarafDev** [comment]
> hey, I'm proficient flutter developer, new to open source. looking for small issues. please assign me this issue.

@khush1yaaar Sure! go ahead

**Comment by khush1yaaar** [comment]
Hey! I tried everything but I'm not able to run this project on my system. 

it's showing this error :- You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply
again and again.

can you help me with that?!

**Comment by AryanSarafDev** [comment]
> Hey! I tried everything but I'm not able to run this project on my system.
> 
> it's showing this error :- You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply again and again.
> 
> can you help me with that?!

Hi! which version of flutter you are using? 
try these solutions - "https://stackoverflow.com/questions/78032396/applying-flutters-app-plugin-loader-gradle-plugin-imperatively-using-the-apply-s"

**Comment by Gaurav-Kushwaha-1225** [comment]
> Hey! I tried everything but I'm not able to run this project on my system.
> 
> it's showing this error :- You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply again and again.
> 
> can you help me with that?!

Update the `kotlin.version` in `/android/build.gradle` file to its latest version.

**Comment by Saurabhsing21** [comment]
Hey @AryanSarafDev, I’d love to contribute to this feature! If the issue is still open, could you please assign it to me? I\

**Comment by Saurabhsing21** [comment]
@AryanSarafDev can you assign me ?

**Comment by Saurabhsing21** [comment]
hey @khush1yaaar  are you working on this ?

**Comment by khush1yaaar** [comment]
Yes, I have submitted a pull request, just a little edit and it's good to go

**Comment by VatsalJain2004** [comment]
Hi , is this issue still open ?? 
else i can fix this..
